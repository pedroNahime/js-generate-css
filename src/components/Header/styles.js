import styled from 'styled-components'

export const Container = styled.header`
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 15px 15px;
  background-color: #585858;
  box-shadow: 0px 5px 8px -5px rgba(88, 88, 88, 1);
`

export const Title = styled.h1`
  display: flex;
  align-items: center;
  font-size: 24px;
  color: white;
  font-weight: bold;
`

export const User = styled.div`
  color: white;
  font-weight: bold;
  display: flex;
  align-items: center;
  font-size: 13px;
  img {
    width: 27px;
    height: 27px;
    border-radius: 50%;
    margin-right: 5px;
  }
`
