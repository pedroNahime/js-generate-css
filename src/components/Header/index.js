import React from 'react'

import { Container, User, Title } from './styles'

const Header = () => (
  <Container>
    <Title>Movie APP</Title>
    <User>
      <img
        src="https://avatars0.githubusercontent.com/u/24299941?s=460&v=4"
        alt="Avatar"
      />
      Pedro Nahime
    </User>
  </Container>
)

export default Header
