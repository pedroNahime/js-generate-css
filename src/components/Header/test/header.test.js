import React from 'react'
import { render } from '@testing-library/react'
import Header from '../index'

describe('<Header/>', () => {
  test('render the component', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
