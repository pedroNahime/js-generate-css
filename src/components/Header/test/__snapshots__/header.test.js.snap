// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> render the component 1`] = `
<div>
  <header
    class="sc-bdVaJa dpMfXo"
  >
    <h1
      class="sc-bwzfXH jcXdsQ"
    >
      Movie APP
    </h1>
    <div
      class="sc-htpNat hmARpE"
    >
      <img
        alt="Avatar"
        src="https://avatars0.githubusercontent.com/u/24299941?s=460&v=4"
      />
      Pedro Nahime
    </div>
  </header>
</div>
`;
